demo/dynamic_embedding/movielens-1m-keras-with-horovod/movielens-1m-keras-with-horovod.py

```diff
@@ -240,6 +241,100 @@ def get_kv_creator(mpi_size: int,
     return de.CuckooHashTableCreator(saver=saver)
 
 
+class DynamicLayerNormalization(LayerNormalization):
```

Comment:
Cool! It is really useful, and I recommend this layer could be a regular API, not only a demo.